xml/rook-caasp-intro.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<chapter xmlns="http://docbook.org/ns/docbook"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0"
+ xml:id="book-intro-rook-caasp">
+<!-- ============================================================== -->
+ <title>Introduction to &ceph; on &casp;</title>
+ <info>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:editurl>https://github.com/SUSE/doc-ses/edit/master/xml/</dm:editurl>
+   <dm:maintainer>asettle@suse.com</dm:maintainer>
+   <dm:status>editing</dm:status>
+   <dm:deadline/>
+   <dm:priority/>
+   <dm:translation>yes</dm:translation>
+   <dm:languages/>
+   <dm:release>SES 7</dm:release>
+  </dm:docmanager>
+ </info>
+ <para>
+   &rook; allows you to run &ceph; on &kube; to provide container
+   workloads all their storage needs. Part of the &productname; family is the
+   &rook; deployment which runs on &casp;.
+ </para>
+ <para>
+   The &rook; deployment is currently only available to nominated
+   and approved customers, it's in limited availability. Contact your
+   SUSE sales team on how to get nominated.
+ </para>
+ <para>
+   &rook; is a so-called storage operator, it automates many steps that you
```

Comment:
Comma splice. I would suggest:

&rook; is a &lt;emphasis&gt;storage operator&lt;/emphasis&gt;. &rook; automates many steps that you